include/z64actor.h

```diff
@@ -100,6 +100,38 @@ typedef struct {
     /* 0x18 */ Vec3f feetPos[2]; // Update by using `Actor_SetFeetPos` in PostLimbDraw
 } ActorShape; // size = 0x30
 
+#define ACTOR_FLAG_NONE 0
+#define ACTOR_FLAG_0 (1 << 0)
+#define ACTOR_FLAG_2 (1 << 2)
+#define ACTOR_FLAG_3 (1 << 3)
+#define ACTOR_FLAG_4 (1 << 4)
+#define ACTOR_FLAG_5 (1 << 5)
+#define ACTOR_FLAG_6 (1 << 6)
+#define ACTOR_FLAG_7 (1 << 7)
+#define ACTOR_FLAG_8 (1 << 8)
+#define ACTOR_FLAG_9 (1 << 9)
+#define ACTOR_FLAG_10 (1 << 10)
+#define ACTOR_FLAG_11 (1 << 11)
+#define ACTOR_FLAG_12 (1 << 12)
+#define ACTOR_FLAG_13 (1 << 13)
+#define ACTOR_FLAG_14 (1 << 14)
+#define ACTOR_FLAG_15 (1 << 15)
+#define ACTOR_FLAG_16 (1 << 16)
+#define ACTOR_FLAG_17 (1 << 17)
+#define ACTOR_FLAG_18 (1 << 18)
+#define ACTOR_FLAG_19 (1 << 19)
+#define ACTOR_FLAG_20 (1 << 20)
+#define ACTOR_FLAG_21 (1 << 21)
+#define ACTOR_FLAG_22 (1 << 22)
+#define ACTOR_FLAG_23 (1 << 23)
+#define ACTOR_FLAG_24 (1 << 24)
+#define ACTOR_FLAG_25 (1 << 25)
+#define ACTOR_FLAG_26 (1 << 26)
+#define ACTOR_FLAG_27 (1 << 27)
+#define ACTOR_FLAG_28 (1 << 28)
+
+#define ACTOR_FLAGS_ALL(flags, mask) (((flags) & (mask)) == (mask))
```

Comment:
Have two issues with this macro

* I feel it's named very poorly. ACTOR_FLAGS_ALL sounds like the set of all flags, instead of something with functionality
* The operation of this macro is fairly generic, which makes me wonder if it should be a general purpose macro instead of one tied to the ACTOR_FLAGs domain

Reply:
`ACTOR_FLAGS_CHECK_ALL` will satisfy the first point
not sure how i feel about a generic macro yet, but its something to discuss

Reply:
I think there can be no confusion with `ACTOR_FLAGS_ALL` being all the flags or'ed together because
1) that wouldn't make sense (there is no use for such a value)
2) imo `if(ACTOR_FLAGS_ALL(thisx->flags, ACTOR_FLAG_1 | ...))` is very readable as is

Would more people want to throw "CHECK_" in there?